intro_to_R_tidyverse/intro_to_r.Rmd

```diff
@@ -0,0 +1,391 @@
+str(nums)
+```
+
+Let's make a *character* vector.
+The `c` function allows us to put multiple things together in one vector.
+We'll make a character vector and call this object `words`.
+```{r Non-numeric vector}
+# Let's store these words as a vector
+words <- c("apple", "boy", "cat", "dog", "elephant")
+
+# What do you think this will give us now?:
+str(words)
+```
+
+Let's make a *factor* vector. 
+If we want R to treat the strings that are the same as the same value such as if
+we had the vector below where `blue` and `red` are repeated, we would need to 
+make this into a `factor.` Side note about naming objects, its best to stay 
+away from names that are already in your R environment as something else. 
+In this case, it might make sense to us to call this vector "colors" but there is
+already a base function in R called `colors`. 
+So to avoid R or us getting confused we'll call this `colorz`. 
+```{r}
+# Let's make a color vector
+colorz <- c("red", "blue", "red", "yellow", "blue")
+```
+  
+Let's take a look at our vector `colorz` and what it looks like.
+```{r}
+str(colorz)
+```
+ Oh no! We wanted to make a factor vector but R thinks that we want a character
+vector. That's because R will, by default, assume this is a character string. 
+
+But what's the difference anyway?...
+  
+Let's take a look at our object another way using the function `summary`.
+Notice that if we use `summary` on a this character vector, it doesn't give us
+terribly meaningful information about how many times each color is repeated 
+because R sees this as a character string and doesn't know to look at the data
+like we want it to. 
+```{r Summary}
+summary(colorz)
+```
+
+If we want `colorz` to be a factor we need to tell R to treat it as such. 
+If you want to try to tell R to reinterpret something as a `factor`, you need to 
+use the function `as.factor`
+*Note* that every character type has it's own `as.___` function eg (`as.numeric`, 
+`as.character`, etc.).
+```{r Coerce Factor}
+# Let's turn our character vector into a factor
+colorz <- as.factor(colorz)
+
+# Character strings that are the same will be included in the same level
+# What do you think this will give us now that we've turned this into a factor?
+summary(colorz)
+```
+
+*Logical* vectors are ones that can help you find out what data meets 
+different criteria.
+You can create logical vectors from using *logical statements* on other vectors.
+`0` means `FALSE` and `1` means `TRUE`
+
+In R, there are a few types of logical expressions: 
+  `==` : Is this equal? (Note that this is different interpretation than a single
+         `=` which is generally used for arguments in functions or also R will 
+         interpret as an assignment)
+  `!=` : Is this not equal? An exclamation point reverses the previous statement.
+  `>` : Is this greater than? 
+  `<` : Is this lesser than? 
+  `>=` : Is this greater than or equal to?
+  `<=` : Is this lesser than or equal to?
+
```

Comment:
![screen shot 2019-02-01 at 10 58 49 am](https://user-images.githubusercontent.com/15315514/52135246-5ee7e280-2613-11e9-9def-b84cf81aae74.png)

Each operator doesn't on a newline :(

Reply:
Fixed that.